common/supplemental/supplementalData.xml

```diff
@@ -5411,9 +5411,23 @@ XXX Code for transations where no currency is involved
 		<parentLocale parent="en_IN" locales="hi_Latn"/>
 		<parentLocale parent="es_419" locales="es_AR es_BO es_BR es_BZ es_CL es_CO es_CR es_CU es_DO es_EC es_GT es_HN es_MX es_NI es_PA es_PE es_PR es_PY es_SV es_US es_UY es_VE"/>
 		<parentLocale parent="fr_HT" locales="ht"/>
-		<parentLocale parent="no" locales="nb nn"/>
+		<parentLocale parent="no" locales="nb nn no_NO"/>
 		<parentLocale parent="pt_PT" locales="pt_AO pt_CH pt_CV pt_FR pt_GQ pt_GW pt_LU pt_MO pt_MZ pt_ST pt_TL"/>
 		<parentLocale parent="zh_Hant_HK" locales="zh_Hant_MO"/>
+		<parentLocale alt="spoken" parent="ar_SA" locales="ars"/>
+	</parentLocales>
+
+	<parentLocales component="brkitr">
+		<parentLocale parent="zh" locales="zh_Hant"/>
+	</parentLocales>
+
+	<parentLocales component="coll">
```

Comment:
Hmmm. There seems to be a general principle here. For collation, probably also for break iteration, we can do a fallback from a non-default script to the language-only locale. No objection to doing what you have here, but we might want to add a general structure later.